packages/@atjson/offset-annotations/src/utils/video-urls.ts

```diff
@@ -229,12 +229,25 @@ function normalizeTwitchURL(url: IUrl) {
   }
 }
 
+function isWirewaxURL(url: IUrl) {
+  return url.host === "embedder.wirewax.com";
+}
```

Comment:
@mattbedell Are there other URLs to transform as well here?

For example, does a `view.wirewax.com` URL make sense to detect?

Reply:
I will reach out to Product to make sure no variations were omitted from the spec